### DISCUSSION TITLE
Blazor data grid

### DISCUSSION THREAD

**Discussion by domenicocanavero** [dotnet/aspnetcore#52837] [Q&A] [answered]
I'm a little bit confused about which Blazor components to use. Mainly now I need a datagrid component, I found:

1. QuickGrid, deployed with Net8, it is official supported by Microsoft
2. Microsoft.FluentUI.AspNetCore.Components with the FluentDataGrid, also supported by Microsoft?
3. Microsoft.Fast.Components.FluentUI.DataGrid, also supported by Microsoft?

In general which are the blazor components supported by Microsoft?

Thanks

**Comment by Sire** [comment, marked as answer]
The Fluent UI datagrid [is using QuickGrid](https://www.fluentui-blazor.net/DataGrid).
Pick Fluent UI if you want to use that framework, otherwise just use QuickGrid.
